egs/mgb2/asr/local/score.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done